sdlbook.c

```diff
@@ -811,7 +846,8 @@ int main(int argc, char **argv) {
 			}
 			if(need_redraw) game_tick(need_redraw);
 		}
-		if(scroll_dist) need_redraw = change_scroll(scroll_dist);
+		if(scroll_dist_v) need_redraw = change_scroll_v(scroll_dist_v);
+		if(scroll_dist_h) need_redraw = change_scroll_h(scroll_dist_h);
```

Comment:
if there's more than one source for need_redraw, you always gotta or them together, just like in the line below

Reply:
OR like so, to not use else statements?:
```c
		if(scroll_dist_h) need_redraw = change_scroll_h(scroll_dist_h);
		if(scroll_dist_v) need_redraw |= change_scroll_v(scroll_dist_v);
```
I did not fully understand need for `need_redraw` afterall, but above change may be unnecessary without "remedy".

Reply:
yeah like that. actually you can use the oring already from the first to the last statement here. 

need_redraw is the mechanism that allows to not blindly repaint 60 times a second, but only when needed, to not have the cpu spin at 100% while you're reading and eating away your battery while annoying your ears with a spinning cpu fan.

Reply:
you must be right because I'm rocking binary on 32MB DDR1 with almost no issue (it takes a hit when trying to change scale), I like also the fact that it takes 0,5s to build `sdlbook` 👍